### PR TITLE
fixed handling of revisions

### PR DESCRIPTION
**Pull request by drallgood**
fixed proper handling of revisions (i.e. updates after the post has been published)

### CODE REVIEW

File: wordpress-github-sync.php [hunk 1]
```diff
@@ -96,12 +96,16 @@ function secret() {
...
-      if ( wp_is_post_revision( $post_id ) || wp_is_post_autosave( $post_id ) )
+      if ( wp_is_post_autosave( $post_id ) )
         return;
 
       if ( ! $this->oauth_token() || ! $this->repository() )
         return;
 
+      // If this is a revision, get real post ID
+      if ( $parent_id = wp_is_post_revision( $post_id ) ) 
+                $post_id = $parent_id;
```
Comment (parkr) [inline wordpress-github-sync.php]: Indentation?

**Comment by mAAdhaTTah** [comment]
This is actually not necessary, as, whenever a post is saved, the post is saved and a revision is saved. So if we update on GitHub when the post AND the revision is saved, it'll actually get "updated" twice, though I believe we check against the currently existing post on GitHub and only update if there's a change. However, this is still duplicated effort, as we have to call against GitHub to see if there are changes twice.

Thanks for the contribution though.

**Comment by drallgood** [comment]
hmm.. I somehow thought this was necessary, because it didn't trigger any pushes when I updated a post.
Maybe, I'm wrong? Might need to check.

**Comment by mAAdhaTTah** [comment]
Odd. Would you mind testing the latest from #42 and letting me know if that works for you?

**Comment by drallgood** [comment]
Tried that. Didn't work.

When I change an already published article in WP, the changes don't show up on github. No error messages either (as far as I can see)

**Comment by benbalter** [comment]
> This is actually not necessary

@mAAdhaTTah that's my understanding of how revisions work. Good catch.

**Pull request closed without merging**